docs/roles/ruby.md

```diff
@@ -10,10 +10,21 @@ Find a list of accepted version with `rbenv install -l`. Default is 2.4.0
 
 # Rails
 
+## Dependencies
+
 Simply add roles `nodejs` and `ruby` in your playbook.yml.
 Note that rails will not be installed unless specified in your Gemfile.
 
 Using mysql or postgres? then include `mysql` or `postgresql` role before `ruby`.
 
-In the box, run `rails server`, then open your browser on http://\<hostname\>:3000.
-See \<hostname\> in your `parameters.yml`.
+## Run server
+
+You have two options. First, in the box, run `rails server` or `puma`, then open your browser on `http://{hostname}:3000`
+
+Second option is to add the nginx role with the rails template:
+
+```
+- { role: nginx, web_directory: "/vagrant/public", site_template: "rails-site.j2", proxy_port: 3000, when: "no_nginx is not defined" }
```

Comment:
Remove the `when: "no_nginx is not defined"`. Apparently this is an option that's only used in the `django` role to not make it depend on nginx if not necessary (and that's not documented anywhere).